src/skin/skincontext.h

```diff
@@ -266,6 +268,18 @@ class SkinContext {
         return m_scaleFactor;
     }
 
+    PredefinedColorMap getCueColorMap(const QDomNode& node) const {
+        PredefinedColorMap colorMap = Color::predefinedColorSet.defaultMap();
+        for (QString colorName : Color::predefinedColorSet.predefinedColorNames()) {
+            QColor representation = selectColor(node, "Cue" + colorName);
+            if (representation.isValid()) {
+                PredefinedColorPointer originalColor = Color::predefinedColorSet.predefinedColorFromName(colorName);
+                colorMap.setRepresentation(originalColor, representation);
```

Comment:
What happens if a skin has no colors set, and one with colours was loaded before. 
Are we back to original colors then?

Reply:
Yes, this is called on each skin reload. If no colors are defined in the skin, the code inside this `if` won't be executed so the default Rgba values for each color will be used.

Actually, this is per color, so a skin can modify only a subset of the Predefined colors.